Jessy/library_challenge.py

```diff
@@ -0,0 +1,220 @@
+print('Question 1:')
+# You are working on a library management system, here are the list books at the library
+books = ['MY OWN WORDS', 'WHITE FRAGILITY', 'THE BODY KEEPS THE SCORE', 'SO YOU WANT TO TALK ABOUT RACE', 'STAMPED FROM THE BEGINNING', 'JUST MERCY', 'BORN A CRIME',
+         'THE WARMTH OF OTHER SUNS', 'THE COLOR OF LAW', 'THE NEW JIM CROW', 'THE TRUTHS WE HOLD', 'SAPIENS', 'BRAIDING SWEETGRASS', "MY GRANDMOTHER'S HANDS", 'ON TYRANNY']
+
+# 1.0
+# What data type is the object 'books'? How do you know?
+print(type(books))
+# from like week one
+
+# 1.1
+# Create a function 'available_books' to print the books list
+def available_books():
+    print(books)
+# Parameters: Not needed for this function
+# Return: Not needed for this function
+
+# 1.2
+# Run the 'available_books' function
+available_books()
+# SInce it already has the print function within it, I dont need to add it for it to print
+
+
+# 1.3
+# Create a function 'check_out' that removes a book from the books list
+def check_out(lst):
```

Comment:
Do what works for you, my point of feedback here is to write your code with clues as to what is going on. Here, the parameter can be "book" or "title" or something similar